recipes/fredapi/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
Can you open an issue or make a PR upstream to add the license file to the source distribution?

Reply:
This is a copy of the same license.

Reply:
I meant the sdist uploaded to PyPI which doesn't include LICENSE file because it is not listed in the MANIFEST.in file

Reply:
Submitted upstream PR https://github.com/mortada/fredapi/pull/22

Reply:
fredapi's sdist (0.4.2) has now been updated with LICENSE file.  Are we ready to merge this PR now?

Reply:
@isuruf ping... thanks for your help

Reply:
Can you remove this file now? Conda-build will get it from the source directory.